packages/openops/src/lib/openops-tables/rows.ts

```diff
@@ -63,96 +63,147 @@ const semaphore = TablesAccessSemaphore.getInstance();
-      url,
-      authenticationHeader,
-    );
+  return executeWithConcurrencyLimit(
+    async () => {
+      const getRowsResult = await makeOpenOpsTablesGet<{ results: any[] }[]>(
+        url,
+        authenticationHeader,
+      );
 
-    return getRowsResult.flatMap((row: any) => row.results);
-  });
+      return getRowsResult.flatMap((row: any) => row.results);
+    },
+    (error) => {
+      logger.error('Error while getting rows:', {
+        error,
+        url,
+        ...getRowsParams,
+      });
+    },
+  );
 }
 
 export async function updateRow(updateRowParams: UpdateRowParams) {
-  return executeWithConcurrencyLimit(async () => {
-    const authenticationHeader = createAxiosHeaders(updateRowParams.token);
-    return await makeOpenOpsTablesPatch(
-      `api/database/rows/table/${updateRowParams.tableId}/${updateRowParams.rowId}/?user_field_names=true`,
-      updateRowParams.fields,
-      authenticationHeader,
-    );
-  });
+  const url = `api/database/rows/table/${updateRowParams.tableId}/${updateRowParams.rowId}/?user_field_names=true`;
+
+  return executeWithConcurrencyLimit(
+    async () => {
+      const authenticationHeader = createAxiosHeaders(updateRowParams.token);
+      return await makeOpenOpsTablesPatch(
+        url,
+        updateRowParams.fields,
+        authenticationHeader,
+      );
+    },
+    (error) => {
+      logger.error('Error while updating row:', {
+        error,
+        url,
+        ...updateRowParams,
+      });
+    },
+  );
 }
 
 export async function upsertRow(upsertRowParams: UpsertRowParams) {
-  return executeWithConcurrencyLimit(async () => {
-    const authenticationHeader = createAxiosHeaders(upsertRowParams.token);
-    return await makeOpenOpsTablesPut(
-      `api/database/rows/table/${upsertRowParams.tableId}/upsert/?user_field_names=true`,
-      upsertRowParams.fields,
-      authenticationHeader,
-    );
-  });
+  const url = `api/database/rows/table/${upsertRowParams.tableId}/upsert/?user_field_names=true`;
+
+  return executeWithConcurrencyLimit(
+    async () => {
+      const authenticationHeader = createAxiosHeaders(upsertRowParams.token);
+      return await makeOpenOpsTablesPut(
+        url,
+        upsertRowParams.fields,
+        authenticationHeader,
+      );
+    },
+    (error) => {
+      logger.error('Error while upserting row:', {
+        error,
+        url,
+        ...upsertRowParams,
+      });
+    },
+  );
 }
 
 export async function addRow(addRowParams: AddRowParams) {
-  return executeWithConcurrencyLimit(async () => {
-    const authenticationHeader = createAxiosHeaders(addRowParams.token);
-    return await makeOpenOpsTablesPost(
-      `api/database/rows/table/${addRowParams.tableId}/?user_field_names=true`,
-      addRowParams.fields,
-      authenticationHeader,
-    );
-  });
+  const url = `api/database/rows/table/${addRowParams.tableId}/?user_field_names=true`;
```

Comment:
took it outside of executeWithConcurrencyLimit, so that I will have the URL for the logs, and it doesn't need to be inside the lock anyway